styles.css

```diff
@@ -1,262 +1,715 @@
-@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@500;600;700&display=swap");
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@200;400;500;600;700&display=swap');
 
-*,
-::before,
-::after {
+* {
+  margin: 0;
+  padding: 0;
   box-sizing: border-box;
-  font-family: "Poppins", sans-serif;
 }
 
-body,
-h1,
-h2,
-h3,
-p {
-  margin: 0;
-  padding: 0;
+:root {
+  --main-purple-color: #6070ff;
+  --secondary-color: #344563;
+  --font-heavey-pruple-color: #172b4d;
+  --light-grey-color: #7a869a;
+  --very-light-puple-color: #ebebff;
 }
 
 body {
-  overflow-x: hidden;
+  background-color: #e5e5e5;
+  position: relative;
+  z-index: -1;
 }
 
-.bg-white {
-  background: #fff;
+.mobile {
+  display: block;
 }
 
-.bg-n-300 {
-  background: #f7f7f9;
+.desktop,
+.desktop-inline {
+  display: none;
 }
 
-.bg-p-50 {
-  background: #ebebff;
+.overflow-none {
+  overflow: hidden;
 }
 
-.clr-n-900 {
-  color: #000;
+/* Toolbar */
+.toolbar-container {
+  background-color: #fff;
+  position: fixed;
+  top: 0;
+  width: 100%;
 }
 
-.clr-n-800 {
-  color: #172b4d;
+.toolbar-content-container {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 90%;
+  margin: 40px auto;
 }
 
-.clr-n-700 {
-  color: #253858;
+.toolbar-brandname {
+  text-decoration: none;
+  font-family: 'Poppins', sans-serif;
+  font-size: 18px;
+  font-weight: 700;
+  line-height: 20px;
+  letter-spacing: 0;
+  text-align: center;
+  color: var(--main-purple-color);
 }
 
-.clr-n-600 {
-  color: #344563;
+.toolbar-brandname-container:hover {
+  transform: scale(1.2);
 }
 
-.clr-n-500 {
-  color: #7a869a;
+.mobile-menu-container {
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100vh;
+  z-index: 4;
+  background-color: #6070ff;
+  opacity: 90%;
+  background-blend-mode: multiply;
+  overflow: hidden;
 }
 
-.clr-p-500 {
-  color: #396df2;
+.mobile-menu-toggler {
+  display: none;
 }
 
-.clr-p-400 {
-  color: #6070ff;
+.menu-icon-container {
+  display: flex;
```

Comment:
- [ ] It is noticed that `display: flex` property is repeated multiple times, Kindly let us stick to the DRY principle by not repeating a property more than it should 🙂. This can be done by having a class called `.flex-row` or `.flex-column` and assigning it wherever possible as a class name in your `html` markup code.
```
.flex-row {
  display: flex;
}

.flex-column {
  display: flex;
  flex-direction: column;
}
```
- And kindly attend to any repetitions in your code 🙂.